partials/scst-scan/_offline-airgap.hbs.md

```diff
@@ -1,3 +1,69 @@
+## <a id="trivy-with-scan-2-0"></a> Using Trivy with Scan 2.0
+
+The Aqua Trivy vulnerability scanner uses two databases to perform vulnerability scans:
+
+1. Vulnerability Database: This database contains the vulnerability information that is used to scan artifacts. This database is built every six hours on [GitHub](https://github.com/aquasecurity/trivy-db) and is distributed via [GitHub Container registry (GHCR)](https://ghcr.io/aquasecurity/trivy-db). 
+
+2. Java Index Database: This database enables Trivy to identify the groupId, artifactId, and version of JAR files. It is built once a day on [GitHub](https://github.com/aquasecurity/trivy-java-db) and distributed via [GitHub Container registry (GHCR)](https://ghcr.io/aquasecurity/trivy-java-db). 
+
+In an online installation of Tanzu Application Platform, these databases are automatically downloaded from GHCR with each scan execution.  In order to enable scanning in an offline environment, these two databases need to be relocated to your private registry, and Trivy needs to be configured to use your private location instead of downloading from GHCR.
+
```

Comment:
Nit: Trivy also provides a Air-Gapped Environment section [here](https://aquasecurity.github.io/trivy/v0.49/docs/advanced/air-gap/) where it details the vulnerability database download, trivy with specify flags in air-gapped environment, and configurations. Should we link this page here and say "for more details about Air-Gapped see aquasec docs"?